### PR TITLE
New term entry for Java Math.hypot()

### LINKED ISSUE #900: [Term Entry] Java Math Methods: `.hypot()`
### Reason/inspiration (optional)

Under `java/concepts/math-methods/terms/`, create a new term entry for the `Math.hypot()` Java method.

- In the `terms/` folder, make a new folder called `hypot`
- Inside the `hypot` folder, create a new markdown file called `hypot.md`
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Also make sure to include an "Example" section with a \```java code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by YanisaHS**
### Description

I added the `.hypot()` Java Math method in `java/concepts/math-methods/terms/`. The file:
1. Defines `.hypot()`
2. Provides the syntax
3. Provides an example

Closes #900 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/hypot/hypot.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+The **`Math.hypot()`** method is used to get the hypotenuse of a right-angled triangle. It returns sqrt(x<sup>2</sup> + y<sup>2</sup>), where _x_ and _y_ are variable type `double`. It does not return intermediate overflow or underflow, which means the `.hypot()` method will not fail due to overflow or underflow of an _x<sup>2</sup>_ or _y<sup>2</sup>_ value.
+
+## Syntax
+
+```pseudo
+Math.hypot(x, y)
+```
+
+The return type for `.hypot()` is `double`. If either _x_ or _y_ parameter is infinite, the result is positive infinity. If neither argument is infinite, but one or both parameters are `NaN`, the result is `NaN`.
```
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/hypot/hypot.md:24]:
```suggestion
The return type for `.hypot()` is `double`. If either the `x` or `y` parameter is infinite, the result is positive infinity. If neither argument is infinite, but one or both parameters are `NaN`, the result is `NaN`.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@YanisaHS I made a first review and only had one comment.

**Comment by YanisaHS** [comment]
@SSwiniarski Thanks for your feedback! Just committed that change 🥳

**Review by SSwiniarski — APPROVED** [review]
@YanisaHS Looks good for a second review.

File: content/java/concepts/math-methods/terms/hypot/hypot.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+Title: '.hypot()'
+Description: 'A method used to get the hypotenuse of a right-angled triangle.'
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/hypot/hypot.md:3]:
```suggestion
Description: 'Returns the hypotenuse of a right-angled triangle.'
```

File: content/java/concepts/math-methods/terms/hypot/hypot.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.hypot()`** method is used to get the hypotenuse of a right-angled triangle. It returns sqrt(x<sup>2</sup> + y<sup>2</sup>), where _x_ and _y_ are variable type `double`. It does not return intermediate overflow or underflow, which means the `.hypot()` method will not fail due to overflow or underflow of an _x<sup>2</sup>_ or _y<sup>2</sup>_ value.
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/hypot/hypot.md:16]:
```suggestion
The **`Math.hypot()`** method is returns the hypotenuse of a right-angled triangle. It returns sqrt(x<sup>2</sup> + y<sup>2</sup>), where `x` and `y` are of type `double`. It does not return intermediate overflow or underflow, which means the `.hypot()` method will not fail due to overflow or underflow of a `x<sup>2</sup>` or `y<sup>2</sup>` value.
```

File: content/java/concepts/math-methods/terms/hypot/hypot.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+The return type for `.hypot()` is `double`. If either the `x` or `y` parameter is infinite, the result is positive infinity. If neither argument is infinite, but one or both parameters are `NaN`, the result is `NaN`.
+
+## Example
+
+The following example demonstrates using `.hypot()` to find the hypotenuse of a right-angled triangle:
+
+```java
+public class Test {
+    public static void main(String args[]) {
```
Comment (yangc95) [inline content/java/concepts/math-methods/terms/hypot/hypot.md:32]: Can we do 2-space indent?

**Review by yangc95 — COMMENTED** [review]
@YanisaHS I went ahead and did a second review!

Please let me know if you have any concerns with any of my comments and tag me once you've looked them over.

File: content/java/concepts/math-methods/terms/hypot/hypot.md (hunk 4 above)
Comment (YanisaHS) [reply content/java/concepts/math-methods/terms/hypot/hypot.md]: Added in a commit below!

**Comment by YanisaHS** [comment]
Thanks for the feedback @yangc95 !! Done! 😃

**Comment by yangc95** [comment]
@YanisaHS live link: https://www.codecademy.com/resources/docs/java/math-methods/hypot

I made some edits prior to merge: [fdb2f2e](https://github.com/Codecademy/docs/pull/1115/commits/fdb2f2e1b942439f38796d2804601339b860a094)

cc: @SSwiniarski @Dusch4593

**Pull request merged**